lightning/src/ln/msgs.rs

```diff
@@ -1605,6 +1605,10 @@ pub trait ChannelMessageHandler : MessageSendEventsProvider {
 	/// If it's `None`, then no particular network chain hash compatibility will be enforced when
 	/// connecting to peers.
 	fn get_chain_hashes(&self) -> Option<Vec<ChainHash>>;
+
+	/// Handles behavior which is triggered when any message from any peer for any handler
+	/// is received.
+	fn handle_message_received(&self);
 }
```

Comment:
Since is this not handling on specific message, maybe let's call this `message_received` and document as:

"Indicates that a message was received from any peer for any handler. Called before the message is passed to the appropriate handler. Useful for indicating that a network connection is active."

Reply:
`message_received` sounds like a better name for the function.
Rename, and update docs in [pr3010.25](https://github.com/shaavan/rust-lightning/commits/pr3010.25)
Thanks for the suggestion!